lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -746,6 +796,21 @@ blockdev_nvme_unmap(struct nvme_blockdev *nbdev, struct spdk_io_channel *ch,
 	return 0;
 }
 
+static int
+blockdev_nvme_smart_read_data(struct nvme_blockdev *nbdev, struct nvme_blockio *bio, void *buf)
+{
+	if (spdk_nvme_ctrlr_cmd_get_log_page(nbdev->ctrlr, SPDK_NVME_LOG_HEALTH_INFORMATION,
+					     SPDK_NVME_GLOBAL_NS_TAG, buf, 512,
+					     queued_done_adminq, bio)) {
+		return -1;
+	} else {
+		/* increment the outstanding admin commands by 1 */
+		nbdev->nvme_dev->outstanding_admin_cmds++;
```

Comment:
This is modifying global data without any synchronization.  The simplest option to fix this would be to turn this into an atomic increment (__sync_fetch_and_add() or similar) and also change the corresponding decrement.